build.sbt

```diff
@@ -336,7 +337,8 @@ lazy val blazeCore = libraryProject("blaze-core")
     description := "Base library for binding blaze to http4s clients and servers",
     startYear := Some(2014),
     libraryDependencies ++= Seq(
-      blazeHttp
+      blazeHttp,
+      crypto.value,
```

Comment:
Blaze should be able to direclty use JDK crypto APIs since it doesn't need to cross-compile to JS.

Reply:
Good point, but I'm not going to do that until the prior comment is resolved.